### PR TITLE
parachute: init at 0.9.1

### PR DESCRIPTION
**Pull request by mjlbach**
###### Motivation for this change
Parachute is a kwin extension that provides a gnome activites like overview with krunner based app launching and window selection.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{ lib, mkDerivation, fetchFromGitHub
+, kcoreaddons, kwindowsystem, plasma-framework, systemsettings }:
+
+mkDerivation rec {
+  pname = "parachute";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "tcorreabr";
+    repo = "parachute";
```
Comment (iblech) [inline pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix:10]: I believe `"${pname}$` is preferred (though I don't really know why, except that it may be shorter).

Comment (mjlbach) [reply pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix:10]: Hmm, I matched the style of all of the other plugins in the directory

Comment (SuperSandro2000) [reply pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix:10]: ``repo = pname`` avoids a string conversion. I have no strong feelings about this and usually don't say anything either way. Just keep it like it is.

File: pkgs/desktops/plasma-5/3rdparty/kwin/scripts/krohnkite.nix [hunk 2]
```diff
@@ -22,10 +22,8 @@ mkDerivation rec {
   # 2. plasmapkg2 doesn't copy metadata.desktop into place, so we do that manually
   installPhase = ''
     runHook preInstall
-
-    plasmapkg2 --type kwinscript --install ${src}/res/ --packageroot $out/share/kwin/scripts
-    install -Dm644 ${src}/res/metadata.desktop $out/share/kservices5/krohnkite.desktop
-
+    plasmapkg2 --type kwinscript --install ${src} --packageroot $out/share/kwin/scripts
```
Comment (SuperSandro2000) [inline pkgs/desktops/plasma-5/3rdparty/kwin/scripts/krohnkite.nix:25]: Can you move this into a separate commit?

File: pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  installPhase = ''
+    runHook preInstall
+    plasmapkg2 --type kwinscript --install ${src} --packageroot $out/share/kwin/scripts
+    install -Dm644 ${src}/metadata.desktop $out/share/kservices5/Parachute.desktop
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Look at your windows and desktops from above.";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix:32]: gpl3Only or gpl3Plus?

File: pkgs/desktops/plasma-5/3rdparty/kwin/scripts/krohnkite.nix (hunk 2 above)
Comment (mjlbach) [reply pkgs/desktops/plasma-5/3rdparty/kwin/scripts/krohnkite.nix:25]: That... should not have happened.

File: pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix (hunk 3 above)
Comment (mjlbach) [reply pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix:32]: I believe [gpl3Only](https://github.com/tcorreabr/Parachute/blob/master/LICENSE). I'll update it.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110430` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages marked as broken and skipped:
</summary>
<ul>
<li>libsForQt512.parachute</li>
<li>libsForQt514.parachute</li>
</ul>
</details>
<details>
<summary>
1 package built:
</summary>
<ul>
<li>libsForQt5.parachute (libsForQt515.parachute ,plasma5Packages.parachute)</li>
</ul>
</details>

**Review by seqizz — COMMENTED** [review]
LGTM but can't approve per-se since I am not using Kwin anymore so can't quickly test it.

**Comment by SuperSandro2000** [comment]
@ttuegel can you take a look at this?

File: pkgs/top-level/aliases.nix [hunk 4]
```diff
@@ -878,6 +878,7 @@ mapAliases ({
     kwin-dynamic-workspaces
     kwin-tiling
     krohnkite
+    parachute
```
Comment (ttuegel) [inline pkgs/top-level/aliases.nix:881]: We usually add aliases for backwards compatibility, so I don't we should add one for a new package.

**Review by ttuegel — CHANGES_REQUESTED** [review]

File: pkgs/top-level/aliases.nix (hunk 4 above)
Comment (mjlbach) [reply pkgs/top-level/aliases.nix]: Fixed, thanks!
